docs/scenarios/library-authoring/trimming.md

```diff
@@ -59,32 +59,32 @@ Next, you restore packages with `dotnet restore`, inspect the `project.lock.json
 Here's what the relevant section in the `project.lock.json` file looks like when targeting `netstandard1.0`:
 
 ```json
-"NETStandard.Library/1.5.0-rc2-24027":{
+"NETStandard.Library/1.6.0":{
     "type": "package",
     "dependencies": {
-        "Microsoft.NETCore.Platforms": "1.0.1-rc2-24027",
-        "Microsoft.NETCore.Runtime": "1.0.2-rc2-24027",
-        "System.Collections": "4.0.11-rc2-24027",
-        "System.Diagnostics.Debug": "4.0.11-rc2-24027",
-        "System.Diagnostics.Tools": "4.0.1-rc2-24027",
-        "System.Globalization": "4.0.11-rc2-24027",
-        "System.IO": "4.1.0-rc2-24027",
-        "System.Linq": "4.1.0-rc2-24027",
-        "System.Net.Primitives": "4.0.11-rc2-24027",
-        "System.ObjectModel": "4.0.12-rc2-24027",
-        "System.Reflection": "4.1.0-rc2-24027",
-        "System.Reflection.Extensions": "4.0.1-rc2-24027",
-        "System.Reflection.Primitives": "4.0.1-rc2-24027",
-        "System.Resources.ResourceManager": "4.0.1-rc2-24027",
-        "System.Runtime": "4.1.0-rc2-24027",
-        "System.Runtime.Extensions": "4.1.0-rc2-24027",
-        "System.Text.Encoding": "4.0.11-rc2-24027",
-        "System.Text.Encoding.Extensions": "4.0.11-rc2-24027",
-        "System.Text.RegularExpressions": "4.0.12-rc2-24027",
-        "System.Threading": "4.0.11-rc2-24027",
-        "System.Threading.Tasks": "4.0.11-rc2-24027",
-        "System.Xml.ReaderWriter": "4.0.11-rc2-24027",
-        "System.Xml.XDocument": "4.0.11-rc2-24027"
+        "Microsoft.NETCore.Platforms": "1.0.1",
+        "Microsoft.NETCore.Runtime": "1.0.2",
+        "System.Collections": "4.0.11",
+        "System.Diagnostics.Debug": "4.0.11",
+        "System.Diagnostics.Tools": "4.0.1",
+        "System.Globalization": "4.0.11",
+        "System.IO": "4.1.0",
+        "System.Linq": "4.1.0",
+        "System.Net.Primitives": "4.0.11",
+        "System.ObjectModel": "4.0.12",
+        "System.Reflection": "4.1.0-",
+        "System.Reflection.Extensions": "4.0.1",
+        "System.Reflection.Primitives": "4.0.1",
+        "System.Resources.ResourceManager": "4.0.1",
+        "System.Runtime": "4.1.0",
+        "System.Runtime.Extensions": "4.1.0",
+        "System.Text.Encoding": "4.0.11",
+        "System.Text.Encoding.Extensions": "4.0.11",
+        "System.Text.RegularExpressions": "4.1.0",
+        "System.Threading": "4.0.11",
+        "System.Threading.Tasks": "4.0.11",
+        "System.Xml.ReaderWriter": "4.0.11",
+        "System.Xml.XDocument": "4.0.1"
```

Comment:
That document says that System.Xml.**Xml**Document is 4.0.1, but that's not used here. What's used is System.Xml.**X**Document, and that list says that one is 4.0.11.